bin/reth/src/args/network_args.rs

```diff
@@ -55,9 +55,16 @@ impl NetworkArgs {
         config: &Config,
         chain_spec: Arc<ChainSpec>,
     ) -> NetworkConfigBuilder {
+        let chain_bootnodes = match chain_spec.chain.try_into() {
+            Ok(Chain::Mainnet) => mainnet_nodes(),
+            Ok(Chain::Goerli) => goerli_nodes(),
+            Ok(Chain::Sepolia) => sepolia_nodes(),
+            _ => mainnet_nodes(),
+        };
+
```

Comment:
I'd like to make this a `Chain` function, similar to

https://github.com/paradigmxyz/reth/blob/5652cb1686dee4d51fab7b951cad953686e3aa0d/crates/primitives/src/chain/mod.rs#L64-L64

Reply:
Since the constants are in `reth_discv4`, I guess those should move to primitives as well?